### PR TITLE
Codecheck bug fix

### PR DESCRIPTION
**Pull request by lilai23**
[Fix issue] https://github.com/huaweicloud/Sermant/issues/1365

【Modification】Fix the codecheck problem in the code

Plugin| Codecheck
-- | --
sermant-flowcontrol| Public or protected method is missing annotation
sermant-flowcontrol| Method header annotation is missing content
sermant-flowcontrol| method is missing annotations for parameters, exceptions or return values
sermant-flowcontrol | Double check lock acquisition singleton should add volatile to the singleton
sermant-flowcontrol| uses custom compareTo but does not override the equals method
sermant-flowcontrol | redundant constructor
sermant-flowcontrol| No handling is done after catching the exception
sermant-flowcontrol| The variable name must conform to the regular expression
sermant-flowcontrol| long type uses L as suffix
sermant-loadbalancer| The method lacks annotations for parameters, exceptions, or return values.
sermant-loadbalancer| Deprecated methods need to add method comments
sermant-loadbalancer| Type force transfer requires verification
sermant-loadbalancer| File header format problem
sermant-monitor| The variable name must conform to the regular expression
sermant-monitor| The method is missing annotations for parameters, exceptions or return values
sermant-monitor| Public or protected method is missing annotation
sermant-monitor| Space indentation problem
sermant-injector| Space indentation problem
sermant-integration-tests| Missing package name
sermant-integration-tests| py file naming problem
sermant-dynamic-config| Variable names must conform to regular expressions

[Use case description] None

[Self-test situation] 1. Local static check passed

[Scope of influence] None

### CODE REVIEW

**Review by hanbingleixue — DISMISSED** [review]

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/handler/retry/AbstractRetry.java [hunk 1]
```diff
@@ -54,7 +57,7 @@ protected final Class<? extends Throwable>[] findClass(String[] classNames) {
                 result.add(Class.forName(className, false, Thread.currentThread().getContextClassLoader()));
             } catch (ClassNotFoundException exception) {
                 LoggerFactory.getLogger().info(String.format(Locale.ENGLISH,
-                    "Can not find retry exception class %s", className));
+                        "Can not find retry exception class %s", className));
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/handler/retry/AbstractRetry.java:60]: 这里应该还是会检查不过吧，需要给日志定一个静态常量

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/inject/DefaultClientHttpResponse.java [hunk 2]
```diff
@@ -80,8 +86,8 @@ public void close() {
         if (responseStream != null) {
             try {
                 responseStream.close();
-            } catch (IOException ignored) {
-                // ignored
+            } catch (IOException e) {
+                LOGGER.log(Level.SEVERE, "Close exception: ", e);
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/inject/DefaultClientHttpResponse.java]: IOException是不是不允许通过日志输出出来

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestInterceptor.java [hunk 3]
```diff
@@ -85,8 +86,8 @@ private Optional<HttpRequestEntity> convertToHttpEntity(Request request) {
                     .setMethod(request.method())
                     .setServiceName(url.getHost())
                     .build());
-        } catch (MalformedURLException ignored) {
-            // ignored
+        } catch (MalformedURLException e) {
+            LOGGER.log(Level.WARNING, "Fail to convert Request to HttpRequestEntity, exception", e);
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestInterceptor.java:90]: 麻烦确认一下，异常信息是否可以通过日志进行打印

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/service/InterceptorSupporter.java [hunk 4]
```diff
@@ -182,8 +195,8 @@ protected final boolean needRetry(io.github.resilience4j.retry.Retry retry, Obje
             try {
                 // 按照第一次等待时间等待
                 Thread.sleep(interval);
-            } catch (InterruptedException ignored) {
-                // ignored
+            } catch (InterruptedException e) {
+                LOGGER.log(Level.WARNING, "Interruption Error:", e);
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/service/InterceptorSupporter.java]: 同上

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/handler/retry/AbstractRetry.java (hunk 1 above)
Comment (lilai23) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/handler/retry/AbstractRetry.java:60]: 这里检查通过的，格式顺带改的

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/inject/DefaultClientHttpResponse.java (hunk 2 above)
Comment (lilai23) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/inject/DefaultClientHttpResponse.java]: IOException是FileNotFoundException的父类，FileNotFoundException是敏感异常，应该不能打印调用栈

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestInterceptor.java (hunk 3 above)
Comment (lilai23) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestInterceptor.java:90]: 非敏感异常

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/service/InterceptorSupporter.java (hunk 4 above)
Comment (lilai23) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/service/InterceptorSupporter.java]: 非敏感异常

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/inject/DefaultClientHttpResponse.java (hunk 2 above)
Comment (lilai23) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/inject/DefaultClientHttpResponse.java]: 已修改

**Review by hanbingleixue — DISMISSED** [review]

**Review by luanwenfei-venus — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Pull request merged**